### ISSUE TITLE
get rid of CouchDB process on Soledad tests

### ISSUE THREAD

**Issue by shyba** [pixelated/pixelated-platform#107]
That's a quick, but necessary improvement for a current (annoying) problem.
Soledad tests currently starts a process for CouchDB.
This shouldn't happen as it is very flaky.
Proposed change will ensure that each db is created, deleted and has a unique name for isolation (for parallelism on test suite).

Definition of done:
trial -j 8 leap.soledad.common.tests
Should run everything without a new process, without flakiness related to isolation and in parallel.

**Comment by shyba** [comment]
https://github.com/leapcode/soledad/pull/269

Removes CouchDB process creation, isolate database names making them unique (at least all I could find so far) and tries to make sure that db is cleaned after each test.

**Comment by shyba** [comment]
269 merged.
270 (pep8) merged.
Snap just failed due to a lower file limit, which indicates an underlying bug on close methods.
That is awesome, I managed to reproduce it here and probably fixed. New PR 271 created.

**Comment by fbernitt** [comment]
:+1: 

```
trial -j 8 leap.soledad.common.tests

...
Ran 906 tests in 131.215s

PASSED (skips=302, successes=640)
```
